go.mod

```diff
@@ -4,14 +4,22 @@ go 1.23.0
 
 require (
 	github.com/cloudbees-io/registry-config v0.0.0-20240808192011-86b0b147c63e
+	github.com/cloudevents/sdk-go/v2 v2.15.2
```

Comment:
the cloudsdk does not need to be included yet. Lets run `go mod tidy` on this.

Reply:
DONE